src/api/draftSpecs.ts

```diff
@@ -5,6 +5,7 @@ import {
     updateSupabase,
 } from 'services/supabase';
 import { ENTITY } from 'types';
+import { CaptureBinding, CaptureDef } from '../../flow_deps/flow';
```

Comment:
Not a fan of this relative pathing from the `src` directory. It can be particularly a pain to use flow types within deeply nested components. How do we feel about updating _tsconfig.json_ and whitelisting this path?

Reply:
Yeah - please add this to tsconfig. We'll also need to make sure that the UI project contains all the docs (or links to docs) needed to run the command to populate those types.